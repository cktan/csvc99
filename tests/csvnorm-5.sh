# Test Case : CSV with null values and empty fields
../csvnorm in/csvnorm-5.csv 
../csvnorm in/csvnorm-5.csv -n "\N"
