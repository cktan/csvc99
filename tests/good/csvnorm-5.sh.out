John,NULL,Boston
John,"",Boston
John,'',Boston
John,\N,Boston
John,NULL,Boston
John,"",Boston
John,'',Boston
John,NULL,Boston
